packages/create-vite/template-lit-ts/tsconfig.json

```diff
@@ -16,9 +16,11 @@
 
     /* Linting */
     "strict": true,
+    "strictBuiltinIteratorReturn": true,
```

Comment:
It looks like this already defaults to true if strict is set to true: https://www.typescriptlang.org/tsconfig/#strictBuiltinIteratorReturn. Maybe we can remove this?

Reply:
Ah, didn't know that. I'll remove this.